README.md

```diff
@@ -33,7 +33,7 @@ Example
 ### First-time setup
 
 ```bash
-go install github.com/loov/goda@v0.4.3
+go install github.com/loov/goda@v0.5.2
```

Comment:
i've been wanting to upgrade `goda` for some time, unfortunately the behavior of its `-cluster` flag changed recently (no longer clusters by directory), and i prefer the old behavior. i believe this changed in https://github.com/loov/goda/pull/60

some examples of the differences:

# goda@v0.4.3

<img width="1404" alt="gographs 0 4 3" src="https://user-images.githubusercontent.com/236915/193514759-10e9dd3a-9852-442b-b70b-71f794058f01.png">

<img width="1392" alt="goda 0 4 3" src="https://user-images.githubusercontent.com/236915/193514756-e8a3fb48-62be-4575-9464-6e9f745be077.png">

# goda@v0.5.2

<img width="1412" alt="gographs 0 5 2" src="https://user-images.githubusercontent.com/236915/193514757-7a5220b4-3eca-41dc-9bf0-88807c5ab07a.png">

<img width="1430" alt="goda 0 5 2" src="https://user-images.githubusercontent.com/236915/193514748-c9c9bff9-85e4-46aa-9f27-00083e152755.png">

Reply:
Ahh okay I see